mne/io/edf/tests/test_gdf.py

```diff
@@ -102,7 +102,9 @@ def test_gdf2_data():
     # gh-5604
     assert raw.info['meas_date'] is None
     _test_raw_reader(read_raw_gdf, input_fname=gdf2_path + '.gdf',
-                     eog=None, misc=None)
+                     eog=None, misc=None,
+                     test_scaling=False,  # XXX this should be True
```

Comment:
you mean it's not working yet on gdf data?

Reply:
The values seem too large. Each of these requires some investigation. I think it should be done in another PR. It's possible nothing is broken but it would be good to understand why the EEG values are around 0.01 even after removing the mean or applying an average reference (for example)